get_query.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * This file contains functions that returns sql queries for GET requests
+ * according to the API documentation.
+ */
+
+/**
+ * Returns sql query that returns an unordered list of questions.
+ *
+ * @return String A sql command that returns an unorderedlist of questions.
+ */
+function get_questions()
+{
+    return "SELECT question_id, answer_type, content FROM questions;";
+}
+
+/**
+ * Returns sql query that returns an unordered list of tas or instructors.
+ * If $ta_list is true, a list of tas is returned. If $ta_list is false,
+ * the returned list depends on column parameter in the url.
+ *
+ * For what=tas, term parameter sepcifies the term returned and course_code
+ * sepcifies the course. term and course_code parameters are optional.
+ *
+ * For what=course_pairings, column parameter is required. term and course_code
+ * are optional parameters.
+ *
+ * @param ta_list True if function call if for list of TAs
+ * @return String A sql command that returns an unorderedlist of questions.
+ */
+function get_course_associations($user_id, $column, $course_code, $term)
+{
+    $specify_course = "";
+    if ($course_code) {
+        $specify_course = "WHERE course_code='$course_code'";
```

Comment:
We should be creating PDO suitable strings, not raw query strings. Raw query strings allow for injection attacks.  See https://stackoverflow.com/questions/60174/how-can-i-prevent-sql-injection-in-php